test/select.test.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var testHelpers = require('./test_helpers');
+
+describe('record selection', function() {
+    var airtable;
+    var testExpressApp;
+    var teardownAsync;
+
+    beforeAll(function() {
+        return testHelpers.getMockEnvironmentAsync().then(function(env) {
+            airtable = env.airtable;
+            testExpressApp = env.testExpressApp;
+            teardownAsync = env.teardownAsync;
+        });
+    });
+
+    afterAll(function() {
+        return teardownAsync();
+    });
+
+    it('selects records without params', function(done) {
+        testExpressApp.set('handler override', function(req, res) {
+            expect(req.method).toBe('GET');
+            expect(req.url).toBe('/v0/app123/Table?');
+            res.json({
+                records: [
+                    {
+                        id: 'recordA',
+                        fields: {Name: 'Rebecca'},
+                        createdTime: '2020-04-20T16:20:00.000Z',
+                    },
+                ],
+                offset: 'offsetABC',
+            });
+        });
+
+        return airtable
+            .base('app123')
+            .table('Table')
+            .select()
+            .eachPage(function page(records) {
+                records.forEach(function(record) {
```

Comment:
This will also pass for any number of records. Could you verify `records.length`?

Reply:
Done.